salt/modules/linux_acl.py

```diff
@@ -132,9 +132,9 @@ def _parse_acl(acl, user, group):
 
     # If a user is not specified, use the owner of the file
     if comps[0] == 'user' and not comps[1]:
-        comps[1] = user
+        comps[1] = 'owner'
```

Comment:
the setfacl command can be used to ways:
1. setfacl u:root:rw /tmp/house/kitchen
2. setfacl u::rw /tmp/house/kitchen
then, there will be two {"user":[{"root": {"octal":6, "permissions":{"read": true,"write": true,"execute": false,}}}]}

if changing to 'owner',it there will be  {"user":[{"root": {"octal":6, "permissions":{"read": true,"write": true,"execute": false,}}}]} and  {"user":[{"owner": {"octal":6, "permissions":{"read": true,"write": true,"execute": false,}}}]}